config/routes.rb

```diff
@@ -418,8 +418,8 @@
     get 'performance/apg', to: 'performance#apgs', as: :apgs
     get 'management', to: 'site#management', as: :management
     get 'events', to: 'events#index', as: :events
-    get 'events/:id', to: 'events#show', as: :event
```

Comment:
While testing all of the code that uses `subscribeExportChannel`, I discovered that `GET /admin/events/export` was broken because it was incorrectly matching `events/:id`. Reordering the routes fixed this.